test/integration/targets/command_shell/tasks/main.yml

```diff
@@ -167,6 +179,32 @@
     that:
       - command_result3 is not changed
 
+# changes
+
+- name: Verify that afile.txt content has changed
+  assert:
+    that: "'win' in lookup('file', '{{ output_dir_test }}/afile.txt') | b64encode | b64decode"
+
+- name: Changes afile.txt content
+  command: "{{ output_dir_test }}/change_afile.sh {{output_dir_test }}/afile.txt"
+  args:
+    changes: "{{ output_dir_test }}/afile.txt"
```

Comment:
The result of this task needs to be registered so that later it can be checked to ensure it reported `change: True`.

Reply:
Also probably a good idea to use a file just for the "changes" tests rather than reusing `afile.txt`.